components/base/Table.tsx

```diff
@@ -0,0 +1,67 @@
+import React from "react";
+import styled from "@emotion/styled";
+import { ROBOTO } from "../typography";
+import { HEADLINE_TEXT, PARAGRAPH_TEXT, GREY_1, GREY_2 } from "../colors";
+
+// Styles for Table
+const StyledTable = styled.table`
```

Comment:
So right now this is rendering:
```
<table>
  <table>
    <tr>
      ...
```

It feels like this should either be a styled div implemented like it is right now, or it should be a styled table where the `<table>` is omitted from the children being passed as props

Reply:
Whoops! Good catch, Connor! 

This is a remnant from when I moved away from the component handling the creating the table itself. I'd say let's change it to a styled div.